db/migrate/20231003124327_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, null: false, foreign_key: { to_table: :users }
```

Comment:
- [ ] According to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) this is supposed to create `user_id` and a corresponding index, for you it creates an `author_id`. Kindly fix that 🙃 🙃

Reply:
@cynthiainga Done making the changes